test/data/jenkins.yaml

```diff
@@ -0,0 +1,253 @@
+jenkins:
+  authorizationStrategy:
+    roleBased:
+      roles:
+        global:
+          - assignments:
+              - admin
+              - admin1
+              - admin2
+            name: admin
+            pattern: .*
+            permissions:
+              - Overall/Administer
+              - Overall/Read
+              - Job/Move
+              - Job/Build
+              - Job/Read
+              - Job/Delete
+              - Job/Create
+              - Job/Discover
+              - Job/Cancel
+              - Job/Configure
+              - Job Config History/DeleteEntry
+              - Job/Workspace
+              - Lockable Resources/View
+              - Lockable Resources/Unlock
+              - Lockable Resources/Reserve
+              - Credentials/Delete
+              - Credentials/ManageDomains
+              - Credentials/Update
+              - Credentials/View
+              - Credentials/Create
+              - Manage ownership/Nodes
+              - Manage ownership/Jobs
+              - Agent/Configure
+              - Agent/Create
+              - Agent/Build
+              - Agent/Provision
+              - Agent/Connect
+              - Agent/Delete
+              - Agent/Disconnect
+              - Run/Replay
+              - Run/Delete
+              - Run/Update
+              - View/Delete
+              - View/Read
+              - View/Create
+              - View/Configure
+              - SCM/Tag
+          - assignments:
+              - anyonomous
+            name: read
+            pattern: .*
+            permissions:
+              - Overall/Read
+              - Job/Read
+  securityRealm:
+    oic:
+      authorizationServerUrl: http://localhost
+      clientId: clientId
+      disableSslVerification: true
+      emailFieldName: emailFieldName
+      escapeHatchEnabled: true
+      escapeHatchGroup: escapeHatchGroup
+      escapeHatchUsername: escapeHatchUsername
+      fullNameFieldName: fullNameFieldName
+      groupsFieldName: groupsFieldName
+      logoutFromOpenidProvider: true
+      scopes: scopes
+      tokenServerUrl: http://localhost
+      userNameField: userNameField
+security:
+  apiToken:
+    creationOfLegacyTokenEnabled: false
+    tokenGenerationOnCreationEnabled: false
+    usageStatisticsEnabled: true
+  copyartifact:
+    mode: PRODUCTION
+  globalJobDslSecurityConfiguration:
+    useScriptSecurity: true
+  sSHD:
+    port: -1
+unclassified:
+  ansiColorBuildWrapper:
```

Comment:
we can remove this ansiColorBuildWrapper as by default this be coming default from ANSI plugin.

Reply:
Removed.